blacklight-cornell/app/helpers/advanced_helper.rb

```diff
@@ -36,9 +36,9 @@ def default_form_values(params)
   end
 
   def params_to_form_values(params)
-    q_row = params[:q_row] || []
-    op_row = params[:op_row] || []
-    search_field_row = params[:search_field_row] || []
+    q_row = Array(params[:q_row]) || []
```

Comment:
If you pass in params like `q_row[]`, instead of `q_row` into the url, you shouldn't have to change this method or the one in CornellParamsHelper - this would be more consistent with how we form the /edit urls elsewhere. Thoughts?